### PR TITLE
[feat][frontend] integrate QR code scanner with dispatch task API

### PR DESCRIPTION
**Pull request by WilhelmWillie**

### CODE REVIEW

File: api/live.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const actions = {
 
 router.post("/dispatch", async (req, res) => {
   const { userId, actionId } = { ...req.body };
+  console.log({ userId, actionId });
```
Comment (saresend) [inline api/live.js]: What's our stance on `console.log`?

**Comment by saresend** [comment]
One quick comment also: In theory each of the actions dispatchers should handle error cases like invalid checkin, so hopefully it's not necessary to have a route to verify an action and then another to actually dispatch it

File: api/live.js (hunk 1 above)
Comment (WilhelmWillie) [reply api/live.js]: oh yeet will remove -- still considering this WIP but will make sure to remove that

**Comment by WilhelmWillie** [comment]
> One quick comment also: In theory each of the actions dispatchers should handle error cases like invalid checkin, so hopefully it's not necessary to have a route to verify an action and then another to actually dispatch it

ooh true -- part of of the reason i have that additional route is so the scanner has some info they can use to verify in-person and compare to whatever ID they have -- this could easily be baked into the return message from /dispatch though

**Comment by WilhelmWillie** [comment]
still consider this PR to be a WIP though so will address those in this PR

**Comment by WilhelmWillie** [comment]
changing my mind -- going to go ahead and merge this in then fix stuff in follow-up PRs

**Pull request merged**